figo/figo.py

```diff
@@ -1113,12 +1094,12 @@ def remove_bank_pin(self, bank_or_bank_id):
         :Parameters:
         - `bank_or_bank_id` - bank whose pin should be removed or its ID
         """
-        if isinstance(bank_or_bank_id, STRING_TYPES):
-            self._request_with_exception("/rest/banks/%s/remove_pin" % bank_or_bank_id,
-                                           method="POST")
-        else:
-            self._request_with_exception("/rest/banks/%s/remove_pin" % bank_or_bank_id.bank_id,
-                                           method="POST")
+        if isinstance(bank_or_bank_id, BankContact):
+            bank_or_bank_id = bank_or_bank_id
+
+        query = "/rest/banks/{0}/remove_pin".format(bank_or_bank_id.bank_id)
```

Comment:
this dosen't work if you get a string ... and the above check should get the `bank_id` attribute and just use `bank_or_bank_id` here